awx/api/templates/api/user_me_list.md

```diff
@@ -6,4 +6,4 @@ One result should be returned containing the following fields:
 
 {% include "api/_result_fields_common.md" %}
 
-Use the primary URL for the user (/api/v1/users/N/) to modify the user.
+Use the primary URL for the user (/api/v2/users/N/) to modify the user.
```

Comment:
well this wasn't ideal documentation. We could peel some of these off for sooner merge.